scrapers/covid-19/getVaccine.js

```diff
@@ -3,14 +3,18 @@ const axiosCookieJarSupport = require('axios-cookiejar-support').default;
 const tough = require('tough-cookie');
 const csv = require('csvtojson');
 const logger = require('../../utils/logger');
+const countries = require('../../utils/countries');
 const { phaseData } = require('../../utils/RAPS');
 
 axiosCookieJarSupport(axios);
 
 const cleanData = (data) => {
```

Comment:
Can you write a short docstring comment explaining what this will do, maybe with an example. It's hard to read code with a semi-complex regex and see exactly what is is going to spit out

Reply:
I didn't write that function. I just formatted it by mistake.